flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1408,5 +1408,10 @@
     },
     "org.srb2.SRB2Kart": {
         "flathub-json-modified-publish-delay": "extra-data"
+    },
+    "com.riverbankcomputing.PyQt.BaseApp": {
+        "flathub-json-modified-publish-delay": "This is a BaseApp",
```

Comment:
Why publish delay would be important for a BaseApp?

Reply:
it shouldn't have one.

Reply:
https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp/pull/9

Reply:
I doubt this will be considered as sufficient explanation given flathub admins stance on (ab)using publish delay time. 2h more or less doesn't change much in this case.

Reply:
Yeah, that's poor excuse. There's nothing special about this BaseApp to have it a shorter publish delay.

Reply:
Why do we need the delay at all? The Updates are tested before merging. So why should we keep the new version of the BaseApp, which may have changes/bugfixes that are really needed (I had such a case with one of my Apps) unpublished on the Buildserver for a few Hours. I just failed to understand the reason for this. There were no complaints about the shorter delay at all.